docker/container.go

```diff
@@ -16,71 +17,60 @@ import (
 	"github.com/docker/docker/pkg/term"
 	"github.com/docker/engine-api/client"
 	"github.com/docker/engine-api/types"
-	"github.com/docker/engine-api/types/container"
 	"github.com/docker/engine-api/types/network"
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/libcompose/config"
 	"github.com/docker/libcompose/labels"
 	"github.com/docker/libcompose/logger"
 	"github.com/docker/libcompose/project"
-	"github.com/docker/libcompose/project/events"
-	util "github.com/docker/libcompose/utils"
-	"github.com/docker/libcompose/yaml"
 )
 
 // Container holds information about a docker container and the service it is tied on.
 type Container struct {
-	name            string
-	serviceName     string
-	projectName     string
-	containerNumber int
-	oneOff          bool
-	eventNotifier   events.Notifier
-	loggerFactory   logger.Factory
-	client          client.APIClient
-
-	// FIXME(vdemeester) Remove this dependency
-	service *Service
+	// FIXME(vdemeester) Replace with ContainerClient with engine-api vendor update
+	client    client.APIClient
+	name      string
+	id        string
+	container *types.ContainerJSON
```

Comment:
This name might be confusing to people who are unfamiliar with the distinction between our `Container` struct and that of engine-api. Just a thought, but maybe we can switch to a more explicit name here? I'll admit, nothing better is coming to mind at the moment...